api/bases/neutron.openstack.org_neutronapis.yaml

```diff
@@ -2250,6 +2250,14 @@ spec:
                     description: CaBundleSecretName - holding the CA certs in a pre-created
                       bundle file
                     type: string
+                  ovndb:
```

Comment:
https://github.com/openstack-k8s-operators/openstack-operator/pull/693 uses "ovn" so will keep it consistent here